docs/questions/query-builder/expressions/week.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Week of year
+description: In Metabase, you can group by week of year in the query builder, or extract the week of year from a date column using a custom expression. Metabase supports multiple algorithms for determining the first week of the year.
+---
+
+# Week of year
+
+Weeks of the year are numbered from 1 to 52, 53, or 54 (depending on how the first week of the year is determined). Metabase supports multiple algorithms for determining the first week of the year.
+
+In Metabase, you can group by week of year in the query builder, or extract the week of year from a date column using a custom expression.
+
+Summarizing by Week of year is useful when making period-over-period comparisons - for example, how the metric in the first week of this year differs from the metric in the first week of last year.
+
+## Summarize by Week of year in the query builder
+
+You can choose to summarize a metric by week of year in the query builder.
+
+1. Select a date field in the **Summarize by** block;
```

Comment:
writing this out explicitly because i remember some customers not finding the additional options